src/main/java/com/google/firebase/auth/Tenant.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2019 Google LLC
```

Comment:
Should this now be 2020? Technically the file was created in 2019 :)

Reply:
Done.